ansible/secrets.dist/dev-vagrant.secrets.yml

```diff
@@ -14,3 +14,7 @@ site_smtp_port: 'INSERT_SMTP_PORT_HERE'
 site_smtp_encryption: 'ssl' # options: tls, ssl, login
 site_smtp_user: 'INSERT_SMTP_USER_HERE'
 site_smtp_password: 'INSERT_SMTP_PASSWORD_HERE'
+
+# Algolia
+site_algolia_app_id: "INSERT_ALGOLIA_APP_ID_HERE"
+site_algolia_api_key: "INSERT_ALGOLIA_API_KEY_HERE"
```

Comment:
Even though it does not change functionality, stick with single quotes (') for consistency. (Multiple times in this PR.)

Reply:
There were some (") quotes inside the same file should I change those too ??